src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```
@@ -0,0 +1,285 @@
+{
+    /**
+     * @var string[]
+     */
+    private $configuration = array();
+
+    /**
+     * @var array
+     */
+    private static $defaultConfiguration = array(
+        'exclude' => array(),
+        'include' => array(),
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->configuration = self::$defaultConfiguration;
+
+            return;
+        }
+
+        foreach ($configuration as $key => $value) {
+            if (!\array_key_exists($key, self::$defaultConfiguration)) {
+                throw new InvalidFixerConfigurationException(
+                    $this->getName(),
+                    \sprintf(
+                        '"%s" is not handled by the fixer.',
+                        $key
+                    )
+                );
+            }
+
+            foreach ($configuration[$key] as $functionName) {
+                if (!\is_string($functionName) || \trim($functionName) === '' || \trim($functionName) !== $functionName) {
+                    throw new InvalidFixerConfigurationException(
+                        $this->getName(),
+                        \sprintf(
+                            'Each element must be a non-empty, trimmed string, got "%s" instead.',
+                            \is_object($functionName) ? \get_class($functionName) : \gettype($functionName)
+                        )
+                    );
+                }
+            }
+        }
+
+        if (empty($configuration)) {
+            throw new InvalidFixerConfigurationException(
+                $this->getName(),
+                'Configuration must define "exclude" and/or "include" as an array.'
+            );
+        }
+
+        if (\array_key_exists('include', $configuration) && \array_key_exists('exclude', $configuration)) {
+            $contradictingFunctions = \array_intersect(
```

if one would set same function name under both keys then yes, I think we should raise an configuration error